doc/DESCRIPTION_fr.md

```diff
@@ -1 +1,3 @@
-Ceci est une fausse description des fonctionalités de l'app
+Un pont entre Matrix et Discord présenté comme un service YunoHost. Les messages, les médias et les notifications sont transférés entre un utilisateur de Discord et un utilisateur de Matrix. Le pont "Mautrix-Discord" consiste en un service synapse app et repose sur postgresql (mysql également disponible). Par conséquent, Synapse pour YunoHost doit être installé au préalable.
```

Comment:
You welcome.
Yunohost app support mysql but if you prefer not mention we can do it.

Reply:
>  Yunohost app support mysql but if you prefer not mention we can do it.

Do you mean the Yunohost app user may switch to mysql instead of postgresql if they want to? (real question, I am not aware of everything Yunohost allows to do)

Reply:
I hadn't understood that, it's possible to package with either mysql or psql. But I don't think it's possible to change at user level. 
We can remove it so ?